docs/src/5.1.MigratingToVersion0.2.x.mdx

```diff
@@ -0,0 +1,91 @@
+# Migrating from 0.1.x to 0.2.x
+
+The 0.2.1 Worldview release brought a new API for managing interactions and a variety of changes to how interactions are done. This is a breaking change - code that worked under 0.1.x may silently fail under 0.2.x. Below is a summary of what changed and how to update your own code.
+
+## Mouse event changes
+
+Previously, the Worldview-level event handler only passed an `objectId`, not the full object being clicked. Now both the Worldview-level event handlers and Command-level event handlers pass the same information about the click: the ray, and an array of objects that contain the object being clicked and the `instanceIndex`, if the clicked object is part of an instanced command.
+
+We now pass an array of objects because stacked object events can now optionally be enabled: passing `enableStackedObjectEvents={true}` to the `Worldview` component allows seeing all objects at the point the user clicked on, instead of just the top-level object.
+```js
+// New type
+type ClickedObject = {
+  object: Object,
+  instanceIndex?: ?number,
+};
+
+export type ReglClickInfo = {
+  ray: Ray,
+  objects: Array<ClickedObject>,
+};
+
+export type MouseHandler = (MouseEvent, ReglClickInfo) => void;
+```
+
+Usage:
+
+```js
+// Old
+<Worldview
+  onClick={(event, { ray, objectId }) => {
+    setSelectedObjectId(objectId);
+  }}>
+  ...
+</Worldview>
+// New
+<Worldview
+  onClick={(event, { ray, objects }) => {
+    setSelectedObjects(objects.map(({ object, instanceIndex }) => object));
+  }}
+  enableStackedObjectEvents>
+  ...
+</Worldview>
+```
+
+## Hitmap changes
+
+How do we detect which object was interacted with? We use a process called picking, in which we generate an off-screen canvas, or "hitmap", to paint our objects into, assign each object a unique color, and then paint each object into this hitmap. We then detect which object was clicked on by testing which color was underneath the cursor.
```

Comment:
Probably intentional, but just noting this duplicates content from the MouseEvents page. Would it be better to just link there so we don't risk them getting out of sync?